flixel/system/frontEnds/HTML5FrontEnd.hx

```diff
@@ -70,4 +128,17 @@ enum FlxBrowser
 	OPERA;
 	UNKNOWN;
 }
+
+enum FlxPlatform
+{
+	WINDOWS;
+	LINUX;
+	UNIX;
+	MAC;
+	ANDROID;
+	BLACKBERRY;
+	WINDOWS_PHONE;
+	IOS;
```

Comment:
It seems `"iPhone"`, `"iPad"` and `"iPod"` all end up as `IOS`. Would it be useful to the able to distuingish them?